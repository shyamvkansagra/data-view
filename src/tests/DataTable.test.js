import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import DataTable from '../datatable';

test("snapshot renders", () => {
	const component = renderer.create(
		<DataTable
			columns={[]}
			rows={[]}
			onRowClick={() => {}}
			onSelectionChange={() => {}}
		/>
	);
	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});