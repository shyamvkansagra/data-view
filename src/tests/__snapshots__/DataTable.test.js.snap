// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot renders 1`] = `
<div>
  <h3>
    Acme Inc. data table
  </h3>
  <table
    cellPadding="0"
    cellSpacing="0"
    className="table-container"
  >
    <thead>
      <tr>
        <th>
          <input
            checked={true}
            onChange={[Function]}
            onClick={[Function]}
            type="checkbox"
          />
        </th>
      </tr>
    </thead>
    <tbody />
  </table>
</div>
`;
